### PR TITLE
ESC-1111 add hidden legend and text formatting

### PR DESCRIPTION
**Pull request by masahmad**

### CODE REVIEW

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html [hunk 1]
```diff
@@ -91,14 +91,19 @@
     }
 
     @H1(messages("undertakingSector.title"), Some("govuk-heading-xl govuk-!-margin-bottom-5"))
```
Comment (habeeb01) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html:93]: I think you were meant to remove the `h1` and the `p` tag and add them as part of the `govukRadios(Radios( ` definition. You would then set `isPageHeading` to true.

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html [hunk 2]
```diff
@@ -90,15 +90,22 @@
...
     @govukRadios(Radios(
       errorMessage = if(form.hasErrors) {Some(ErrorMessage(
         content = Text(messages("undertakingSector.error.required"))
       ))} else None,
       fieldset = Some(Fieldset(
-        describedBy = Some("sector-desc")
+        describedBy = Some("sector-desc"),
+        legend = Some(Legend(
+        content = Text(messages("undertakingSector.title")),
+        classes = "govuk-fieldset__legend--xl break-word",
```
Comment (habeeb01) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html:103]: please remove `break-word` as it is not required

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html (hunk 1 above)
Comment (masahmad) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html]: done

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html (hunk 2 above)
Comment (masahmad) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html]: updated

**Review by habeeb01 — APPROVED** [review]

**Pull request merged**